voila/handler.py

```diff
@@ -140,17 +145,36 @@ def fix_notebook(self, notebook):
         notebook.metadata.kernelspec.language = all_kernel_specs[kernel_name]['spec']['language']
         raise tornado.gen.Return(notebook)  # TODO py2: replace by return
 
+    @tornado.gen.coroutine
+    def create_notebook(self, model, language):
+        all_kernel_specs = yield tornado.gen.maybe_future(self.kernel_spec_manager.get_all_specs())
+        kernel_name = yield self.find_kernel_name_for_language(language, kernel_specs=all_kernel_specs)
+        spec = all_kernel_specs[kernel_name]
+        notebook = nbformat.v4.new_notebook(
+            metadata={
+                'kernelspec': {
+                    'display_name': spec['spec']['display_name'],
+                    'language': spec['spec']['language'],
+                    'name': kernel_name
+                }
+            },
+            cells=[nbformat.v4.new_code_cell(model['content'])],
```

Comment:
Maybe later after adding support for the progressive rendering (https://github.com/QuantStack/voila/pull/133) there could be more than one cell created, from cell delimiters such as `%%` or `# In[1]:`.

Reply:
Yes, this could again be a configurable, mapping from file extension to regex. I'd say not now right?

Reply:
Indeed let's do that later (we can open an issue to track it).